src/main.rs

```diff
@@ -283,6 +292,10 @@ async fn main() {
 	// Handle about pages
 	app.at("/about").get(|req| error(req, "About pages aren't added yet".to_string()).boxed());
 
+	// Instance info page
+	app.at("/instance-info").get(|r| instance_info::instance_info(r).boxed());
+	app.at("/instance-info.:extension").get(|r| instance_info::instance_info(r).boxed());
```

Comment:
Consider handling invalid extensions. I just tried `/instance-info.jsn` by accident and got the HTML page, when we really should be returning a 404.